src/HOCs/FirebaseProvider.tsx

```diff
@@ -11,8 +11,8 @@
 
 import * as React from 'react';
 
-import { BannerConsumerProps } from 'HOCs/BannerProvider';
-import { showAccountVerificationBanner } from 'utils/bannerUtility';
+import { WorkBannerConsumerProps } from 'HOCs/WorkBannerProvider';
```

Comment:
Does this need to be prefixed `Work` if there is only one now?

Reply:
Good point! Will remove `Work` from the Banner